ktorm-jackson/src/main/kotlin/org/ktorm/jackson/JsonSqlType.kt

```diff
@@ -31,7 +31,7 @@ import java.sql.Types
  */
 public val sharedObjectMapper: ObjectMapper = ObjectMapper()
     .registerModule(KtormModule())
-    .registerModule(KotlinModule())
+    .registerModule(kotlinModule())
```

Comment:
`kotlinModule()` is a short hand that does the same.

https://github.com/FasterXML/jackson-module-kotlin/blob/2.17/src/main/kotlin/com/fasterxml/jackson/module/kotlin/Extensions.kt#L24-L28

Reply:
I think `koltinModule()` does exactly that: https://github.com/FasterXML/jackson-module-kotlin/blob/2.17/src/main/kotlin/com/fasterxml/jackson/module/kotlin/Extensions.kt#L24

Reply:
@arnodenuijl , could you check the answer from @k163377 and review again?

Reply:
Hi. Sadly it's not mine to merge. I just commented because I had the same issue and didn't understand the fix. But it's not my repo and I don't have any rights on it.